core/src/main/java/bisq/core/trade/TradeManager.java

```diff
@@ -297,6 +297,15 @@ private void initPendingTrades() {
                                 trade.getId());
                         tradesWithoutDepositTx.add(trade);
                     }
+
+            try {
+                DonationAddressValidation.validate(trade.getDelayedPayoutTx(),
+                        daoFacade,
+                        btcWalletService);
```

Comment:
```suggestion
                        if (trade.getDelayedPayoutTx() != null) {
                            DonationAddressValidation.validate(trade.getDelayedPayoutTx(),
                                    daoFacade,
                                    btcWalletService);
                        }
```

Reply:
The delayedPayoutTx should be available any time. Did you get some error in logs? Can you reproduce it? Was it on regtest or mainnet?

Reply:
Regtest.

You get that if you test this PR. It will create a trade that doesn't have a delayedPayoutTx